packages/components/src/button/stories/index.js

```diff
@@ -23,142 +18,62 @@ import Button from '../';
 export default {
 	title: 'Components/Button',
 	component: Button,
-	parameters: {
-		knobs: { disable: false },
+	argTypes: {
```

Comment:
In order to make the Storybook example more complete, we could look into adding all of `Button` props — from what I can see, those are missing from the controls:

 - href,
 - target,
 - isPressed,
 - iconPosition (which has a default of `'left'`)
 - showTooltip,
 - tooltipPosition,
 - shortcut,
 - text,
 - describedBy,

Reply:
Is there a plan to deprecate the rendering button as a link (`href` and `target` props)?

Reply:
We still have to take a proper look at Button, but we'd definitely like to do so. The difficulty there is that Button is a very widely used component, and so we need to be extra careful.

The first step in that direction will be to refactor Button to TypeScript

Reply:
I added most of the missing props.

I omitted `href` and `target`. I don't think we want to promote this usage.

The `describedBy` isn't documented in the readme. It might be worth doing a separate follow-up.

Reply:
> I omitted `href` and `target`. I don't think we want to promote this usage.

While I agree that those props aren't really aligned the way we design component APIs nowadays, they are currently documented in the README and therefore I believe that we should list them in Storybook too.

> The `describedBy` isn't documented in the readme. It might be worth doing a separate follow-up.

Happy to leave this one out for the time being and work on it in a follow-up as you suggest.

It's also worth noting that the Storybook docs will be auto-generated from the types once this component will be migrated to TypeScript, and that therefore all prop will be picked up by Storybook docs (unless we explicitly exclude them).